test/contconic.jl

```diff
@@ -0,0 +1,67 @@
+function set_lin1test_solutions!(solver)
+    # SOLVE 1
+    LQOI.set_termination_status!(solver, MOI.Success)
+    LQOI.set_primal_status!(solver, MOI.FeasiblePoint)
+    LQOI.set_dual_status!(solver, MOI.FeasiblePoint)
+
+    LQOI.set_variable_primal_solution!(solver, [1.0, 0.0, 2.0])
+    LQOI.set_constraint_primal_solution!(solver, [1.0, 0.0, 2.0, 3.0, 2.0])
+    LQOI.set_quadratic_primal_solution!(solver, Float64[])
+
+    LQOI.set_variable_dual_solution!(solver, [0.0, 0.0, 0.0])
+    LQOI.set_constraint_dual_solution!(solver, [-0.0, 2.0, -0.0, -3.0, -1.0])
+    LQOI.set_quadratic_dual_solution!(solver, Float64[])

```

Comment:
There must be a nice way to reduce this verbosity...

Reply:
do you mean the function name?

Reply:
No something like
```julia
function set_solution(solver;
                      variable_dual_solution = Float64[],
                      quadratic_dual_solution = Float64[])
    LQOI.set_variable_dual_solution!(solver, variable_dual_solution)
    LQOI.set_quadratic_dual_solution!(solver, variable_dual_solution)
end

set_solution(solver;
             variable_dual_solution = [0.0, 0.0, 0.0])
```

Reply:
I’d be ok with something along those lines.
I’d go for:
set_solution(solver; variable_dual = [0.0, 0.0, 0.0])